README.md

```diff
@@ -1,90 +1,100 @@
-# meta
+# meta-dt
+A reworking of the main meta project to add additional package management facilities, similar to utilities like apt, dpkg, yum, yast2, emerge .... just written in pure shell script (and a tiny bit of REXX) to remove any pre-reqs (like python/perl/bash etc).
+This fork is designed for everyday usage of the zos Open Tools ports within the USS environment or for those who wish to download the tools; users who are interested in building and/or porting packages from scratch should use the main release of meta at this time.
 
-Meta repository to tie together the various underlying z/OS Open Source tools repositories here.
+## Installation
+- Download the pax to a suitable location (for example /tmp).
+- Expand the pax using the command ```pax -rvf <filename>.pax```.  This will expand the pax to the current directory, listing the various included files as it does so.
+- From the ```meta-<ver>/bin``` directory run the following command, answering the questions appropriately:
+```bash
+. ./.env
+zopen init
+```
 
-View our documentation at https://zosopentools.github.io/meta/.
 
-## Background
+## Pre-config
+It is advised to have the following set on the system to ensure correct operation:
+```
+export _BPXK_AUTOCVT=ON
+export _CEE_RUNOPTS="$_CEE_RUNOPTS FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)"
+export _TAG_REDIR_ERR=txt
+export _TAG_REDIR_IN=txt
+export _TAG_REDIR_OUT=txt
+```
 
-There are some key 'foundational' Open Source technologies needed to port software. The goal of this set of repositories is to provide minimal 'port' repositories
-that can be used to get a foundational software package building on z/OS.
-We are starting with what we view as some 'foundational' technologies. One is the stack of technology to be able to build [zsh](https://sourceforge.net/projects/zsh/postdownload). Another is to
-be able to build [protocol buffers](https://github.com/protocolbuffers/protobuf/releases).
+## Sample usage
+```bash
+. ./.env # Source the .env from meta
+zopen init
+. <zopen_root_path>/etc/.zopen-config
+zopen list --installed
+zopen install which
+zopen list --installed
+which which
+zopen upgrade
+```
 
-But - there is a _transitive closure_ problem to address with porting a software package, namely understanding what packages are pre-requisites
-for the software package you want to port. As an example, for zsh, we see the following:
+## Important usage notes
+- On first run, ```zopen init``` will copy this forked version of meta into the "package" area of zopen (ie. where packages are expanded and accessed from).  It will also be pinned to this release to prevent any possible updates from the "real" meta package. Removing the .pinned file from the meta-dt directory will allow for the main meta port to be installed however this will cause incompatabilities if run.
```

Comment:
Is this still right now that it isn’t a fork?

Reply:
Updated